src/Tribe/Main.php

```diff
@@ -3933,24 +3933,24 @@ public function get_closest_event( $post, $mode = 'next' ) {
 			$start_date = get_post_meta( $post->ID, '_EventStartDate', true );
 			$this->_where_start_date = $start_date;
 
-			$args       = array(
-				'post__not_in'   => array( $post->ID ),
-				'meta_key'       => '_EventStartDate',
-				'orderby'        => array( '_EventStartDate' => $order, 'ID' => $order ),
+			$args = [
+				'post__not_in'   => [ $post->ID ],
+				'orderby'        => [ 'start_date_clause' => $order, 'ID' => $order ],
 				'posts_per_page' => 1,
-				'meta_query'     => array(
-					array(
+				'meta_query'     => [
```

Comment:
:warning: Detected usage of meta_query, possible slow query.

<sub>hash: b8cfb96b960ab0cbbc5f3930695de204</sub>